Bindings/Packages/com.virtualmaker.bindings/Runtime/Bindings.GameObject.cs

```diff
@@ -24,5 +25,67 @@ public void BindActive<T>(Component component, IProperty<T> prop, Func<T, bool>
         {
             Bind(prop, value => component.gameObject.SetActive(transformer(value)));
         }
+
+        public void BindList<FromType, ToType>(Transform parent, ToType prefab, IProperty<List<FromType>> prop, Action<FromType, ToType> onTemplateAdded) where ToType : Component
```

Comment:
prefab?

Reply:
Yes. This is the template object that copies are instantiated of and then changed to match the content of the specific element in the list.

Lines 59–61:
```csharp
toItem = GameObject.Instantiate(prefab, parent);
childItems.Add(item, toItem);
onTemplateAdded(item, toItem);
```

Reply:
Maybe ToType should actually be MonoBehaviour then, bc a prefab has a lot more than just Component

Reply:
either that or the name should be `component` not `prefab`